corehq/apps/settings/views.py

```diff
@@ -1,5 +1,5 @@
 from django.views.decorators.debug import sensitive_post_parameters
-from corehq.apps.style.decorators import check_preview_bootstrap3, use_select2
+from corehq.apps.style.decorators import use_bootstrap3, use_select2
```

Comment:
To do the uninstall you can add it to the
requirements/uninstall-requirements.txt file which gets processed during
deploy.